BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/BlobStorageServiceCollectionExtensions.cs

```diff
@@ -24,13 +26,18 @@ public static void AddBlobStorage(this IServiceCollection services, BlobStorageO
         {
             case AZURE_CLOUD_PROVIDER:
                 services.AddAzureStorageAccount(azureStorageAccountOptions => { azureStorageAccountOptions.ConnectionString = options.ConnectionInfo!; });
+                services.AddHealthChecks().AddAzureBlobStorage(
+                    clientFactory: _ => new BlobServiceClient(options.ConnectionInfo),
+                    optionsFactory: _ => new AzureBlobStorageHealthCheckOptions { ContainerName = options.Container }
```

Comment:
```suggestion
                    optionsFactory: sp => new AzureBlobStorageHealthCheckOptions { ContainerName = sp.GetRequiredService<IOptions<BlobOptions>>().Value.RootFolder }
```

Reply:
`BlobOptions` is a part of the `Files.Application` project. Should I create a dependency (Since this is a common project)?

Reply:
Damn. No, no dependency. We have to find another way then. 😞

Reply:
I use `sp.GetRequiredService<IOptions<BlobStorageOptions>>().Value.Container` now